classes/xt_rcar.bbclass

```diff
@@ -0,0 +1,8 @@
+rcar_unpack_proprietary_graphics() {
```

Comment:
Wow! Should we have whole `.bbclass` for renesas quirks? Isn't `.inc` file not enough?